### PR TITLE
fix(component/list/TFD-1033): keep columns with empty list

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

keeping the display of columns even if no items to display

**What is the chosen solution to this problem?**

move the message inside the rendering of Table display

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: packages/components/src/List/DisplayTable/DisplayTable.component.js [hunk 1]
```diff
@@ -304,6 +304,11 @@ function DisplayTable(props) {
 		'tc-list-display-table',
 		theme.table,
 	);
+	let message = null;
+	if (!items || items.length === 0) {
```
Comment (ncomont) [inline packages/components/src/List/DisplayTable/DisplayTable.component.js]: `items.length === 0` is useless here.

File: packages/components/src/List/DisplayTable/DisplayTable.component.js [hunk 2]
```diff
@@ -304,6 +304,11 @@ function DisplayTable(props) {
...
+		message = (<span className={theme['no-result']}>No result found</span>);
```
Comment (ncomont) [inline packages/components/src/List/DisplayTable/DisplayTable.component.js]: Parenthesis are not needed for single line JSX.

File: packages/components/src/List/DisplayTable/DisplayTable.component.js (hunk 1 above)
Comment (jmfrancois) [inline packages/components/src/List/DisplayTable/DisplayTable.component.js]:
```
> !![]
true
> !![].length
false
```
oh you mean === 0 is useless , thanks

Comment (jmfrancois) [inline packages/components/src/List/DisplayTable/DisplayTable.component.js]:
```![]
false
[].length === 0 
true```

Comment (ncomont) [inline packages/components/src/List/DisplayTable/DisplayTable.component.js]: Oh, sorry, I had read `!items.length || items.length === 0` :)

**Pull request merged**